cookbook-react/src/recipes/buttons/zecat/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+$denim: #0B43D1;
+$smalt: #052E91;
+$white: #FFF;
+$black: #000;
+$mercury: #E1E1E1;
+
+.button {
+  border-radius: 5px;
+  color: $white;
+  font-size: 16px;
+  min-width: 200px; // Just for presentational purposes, remove in real project
+  padding: 9px 25px;
+
+  &.primary {
+    background-color: $denim;
+
+    &:active {
+      background-color: $smalt;
+    }
+
+    &:disabled {
+      background-color: $mercury;
+    }
+  }
+
+  &.secondary {
+    background-color: none;
+    border: 1px solid $denim;
+    color: $denim;
+
+    &:disabled {
+      border: 1px solid $mercury;
+      color: $mercury;
+    }
+  }
+
+  &:hover:not(:disabled) {
+    // stylelint-disable-next-line length-zero-no-unit
```

Comment:
What's the issue with this rule?

Reply:
I thought the 0.2 value for the alpha in rgba apparently bothers the linter, but now I just realized it was the "0px". Already fixed it, my bad